padd.sh

```diff
@@ -1140,7 +1140,11 @@ PrintDashboard() {
         moveXOffset; printf " %-10s%-16s %-4s%-7s %-4s%-5s${clear_line}\n" "Interfce:" "${iface_name}" "TX:" "${tx_bytes}" "RX:" "${rx_bytes}"
         moveXOffset; printf " %-10s%-16s %-8s${dnssec_heatmap}%-16s${reset_text}${clear_line}\n" "DNS:" "${dns_information}" "DNSSEC:" "${dnssec_status}"
         moveXOffset; printf " %-10s%s${clear_line}\n" "IPv6:" "${pi_ip6_addr}"
-        moveXOffset; printf " %-10s%-15s%-4s${dhcp_range_heatmap}%-36s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_check_box}" "Rng" "${dhcp_range}"
+        if [ "${dhcp_range}" = "N/A" ]; then
+            moveXOffset; printf " %-10s${dhcp_check_box}  ${blue_text}%-35s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_range}"
+        else
+            moveXOffset; printf " %-10s${dhcp_check_box}  ${dhcp_range_heatmap}%-35s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_range}"
+        fi
```

Comment:
I still think we should simply remove the `"Rng"` string and left the rest as it is, to avoid increasing the code complexity:

```suggestion
        moveXOffset; printf " %-10s%-15s${dhcp_range_heatmap}%-36s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_check_box}" "${dhcp_range}"
```